### PR TITLE
feat: helm lint exposed in Gradle and Maven plugins

### PR DESCRIPTION
**Pull request by manusa**
## Description

Part of https://github.com/eclipse/jkube/issues/2613

Exposed Helm linting capabilities in the Gradle and Maven plugins for OpenShift and Kubernetes.

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.dev/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [x] My code follows the style guidelines of this project
 - [x] I Keep It Small and Simple: The smaller the PR is, the easier it is to review and have it merged
 - [x] I use [conventional commits](https://www.conventionalcommits.org/) in my commit messages
 - [x] I have performed a self-review of my code
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [x] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [x] I have added tests that prove my fix is effective or that my feature works
 - [x] New and existing unit tests pass locally with my changes
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by manusa** [comment]
### Eclipse JKube [CI Report](https://github.com/jkubeio/ci)

Started new GH workflow run for https://github.com/eclipse/jkube/pull/2626 (_2024-02-06T12:03:46Z_)

:gear: [JKube E2E Tests (7799328974)](https://github.com/jkubeio/ci/actions/runs/7799328974) :hourglass_flowing_sand:
- :hourglass_flowing_sand: [Bootstrap CI test run (#2626)](https://github.com/jkubeio/ci/actions/runs/7799328974/job/21269905713)

File: gradle-plugin/openshift/src/test/java/org/eclipse/jkube/gradle/plugin/task/OpenShiftHelmLintTaskTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  }
+
+  @AfterEach
+  void tearDown() {
+    System.clearProperty("jkube.kubernetesTemplate");
+  }
+
+  @Test
+  void runTask_withMissingHelmPackage_shouldThrowException() {
+    KubernetesHelmLintTask kubernetesHelmLintTask = new KubernetesHelmLintTask(KubernetesExtension.class);
```
Comment (rohanKanojia) [inline gradle-plugin/openshift/src/test/java/org/eclipse/jkube/gradle/plugin/task/OpenShiftHelmLintTaskTest.java:55]: Umm, why are we creating KubernetesHelmLinkTask in OpenShiftHelmLinkTaskTest? Probably you copy pasted it from KubernetesHelmLinkTaskTest and forgot to replace it.

Comment (manusa) [reply gradle-plugin/openshift/src/test/java/org/eclipse/jkube/gradle/plugin/task/OpenShiftHelmLintTaskTest.java:55]: yes, copied, ran the tests and forgot to update :facepalm:

Comment (manusa) [reply gradle-plugin/openshift/src/test/java/org/eclipse/jkube/gradle/plugin/task/OpenShiftHelmLintTaskTest.java]: fixed, thx

**Review by rohanKanojia — APPROVED** [review]

**Pull request merged**